src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/drive/DrivingDeltas.java

```diff
@@ -0,0 +1,32 @@
+package org.usfirst.frc1745.deepspace2019.subsystems.drive;
+
+public class DrivingDeltas {
```

Comment:
introduced this object so we don't return a double array when we want to pass around drive numbers. taking suggestions on naming